src/lib/snack-bar/snack-bar-container.ts

```diff
@@ -81,6 +87,14 @@ export class MdSnackBarContainer extends BasePortalHost implements OnDestroy {
     super();
   }
 
+  /**
+   * Gets the current animation state both combining one of the possibilities from
+   * SnackBarState and the vertical location.
+   */
+  getAnimationState(): string {
+    return `${this._animationState}-${this.snackBarConfig.verticalPosition}`;
```

Comment:
Sometimes this is called when `this.snackBarConfig` is still undefined, so tests are failing with 

```
Cannot read property 'verticalPosition' of undefined
```

Add a check?